lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -79,82 +80,57 @@ def expects_uri?
           term_config.term_id_expects_uri?
         end
 
-        def extract_uri_from_id(id)
-          return @uri = id if expects_uri?
+        def uri
+          return @uri if @uri.present?
```

Comment:
Since what we really want is the uri, renamed this method to be consistent with ruby standard practice.